terraform/testcases/containerinsight_eks/parameters.tfvars

```diff
@@ -1,2 +1,4 @@
 # this file is defined in validator/src/main/resources/validations
-validation_config = "eks-container-insight.yml"
\ No newline at end of file
+validation_config = "eks-container-insight.yml"
+
+aoc_base = "infra"
```

Comment:
i seems don't find "infra" used any places?

Reply:
It is used to overwrite the default value of `otlp` to avoid creating otlp related resources.